Firestore/owlbot.py

```diff
@@ -14,52 +14,45 @@
 
 """This script is used to synthesize generated parts of this library."""
 
+import logging
+from pathlib import Path
 import subprocess
+
 import synthtool as s
-import synthtool.gcp as gcp
-import logging
+from synthtool.languages import php
+from synthtool import _tracked_paths
 
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICBazel()
-common = gcp.CommonTemplates()
-
-for ver in ['V1']:
-    lower_version = ver.lower()
+src = Path(f"../{php.STAGING_DIR}/Firestore").resolve()
+dest = Path().resolve()
 
-    library = gapic.php_library(
-        service='firestore',
-        version=lower_version,
-        bazel_target=f'//google/firestore/{lower_version}:google-cloud-firestore-{lower_version}-php',
-    )
+# Added so that we can pass copy_excludes in the owlbot_main() call
+_tracked_paths.add(src)
 
-    # copy all src except partial veneer classes
-    s.move(library / f'src/{ver}/Gapic')
-    s.move(library / f'src/{ver}/resources')
-
-    # copy proto files to src also
-    s.move(library / f'proto/src/Google/Cloud/Firestore', f'src/')
-    s.move(library / f'tests/')
+# Exclude gapic_metadata.json and the partial veneer
+php.owlbot_main(
+    src=src,
+    dest=dest,
+    copy_excludes=[
+        src / "*/src/*/gapic_metadata.json",
+        src / "*/src/*/*.php"
```

Comment:
Do we want to make this a little more specific, like `"*/src/*/*Client.php"`?

Reply:
Sure

Reply:
Done